tests/drivers/pwm/pwm_api/src/test_pwm.c

```diff
@@ -185,4 +187,5 @@ void test_pwm_cycle(void)
 	/* Period : Pulse (64000 : 0), unit (cycle). Voltage : 0V */
 	zassert_true(test_task(DEFAULT_PWM_PORT, DEFAULT_PERIOD_CYCLE,
 				0, UNIT_CYCLES) == TC_PASS, NULL);
+
```

Comment:
Whitespace nit

Reply:
Thank you for spotting this. I removed this change.